lib/index.js

```diff
@@ -15,22 +15,23 @@ const MATCH_URL_ALL = /url\(\s*(['"]?)([^ '"()]+)(\1)\s*\)/g
 const MATCH_IMPORTS = /@import\s+(['"])([^,;'"]+)(\1)(\s*,\s*(['"])([^,;'"]+)(\1))*\s*;/g
 const MATCH_FILES = /(['"])([^,;'"]+)(\1)/g
 
-function getImportsToResolve (original, includePaths) {
+function getImportsToResolve (original, includePaths, transformers) {
   let extname = path.extname(original)
   let basename = path.basename(original, extname)
   let dirname = path.dirname(original)
 
   let imports = []
   let names = [basename]
   let exts = [extname]
+  let extensionPrecedence = [].concat(EXT_PRECEDENCE, Object.keys(transformers))
```

Comment:
I would suggest renaming `extensionPrecedence` to something like `extendedExtensionPrecedences`

Reply:
Also could you go all ES6 on this `let extensionPrecedence = [...EXT_PRECEDENCE, ...Object.keys(transformers)]`

Reply:
Spread syntax will break test on node 4